src/site-server.ts

```diff
@@ -189,4 +190,26 @@ export class SiteServer {
 			return { stdout: '', stderr: 'error when executing wp-cli command', exitCode: 1 };
 		}
 	}
+
+	async isSQLiteInstalled(): Promise< boolean > {
+		const sqlitePath = nodePath.join(
+			this.details.path,
+			'wp-content',
+			'mu-plugins',
+			SQLITE_FILENAME
+		);
+		const wpConfigPath = nodePath.join( this.details.path, 'wp-config.php' );
+		const databaseIndexPath = nodePath.join( this.details.path, 'wp-content', 'db.php' );
+		const databasePath = nodePath.join( this.details.path, 'wp-content', 'database', '.ht.sqlite' );
```

Comment:
I don't think we need to check this because even if you remove this file, the site will still work and use SQLite. It will just create a new .ht.sqlite file automatically.

Reply:
Ah, thanks for pointing this out. I've tested removing the file and confirmed it's recreated. Hence, I'll remove this check.